src/CMakeLists.txt

```diff
@@ -196,10 +198,14 @@ set(headers
   VRPAK.H
   WFPAK.H
   XIOPAK.H
-  XMODULE.h
   YACAM.H
 )
 
+if(USE_XMODULE)
+  list(APPEND source XMODULE.CPP) # Runtime Share Library interface - Not used since Version 0.899. (Windows only)
+  list(APPEND headers XMODULE.H) # Runtime Share Library interface - Not used since Version 0.899. (Windows only)
```

Comment:
Probably don't need to repeat the comment twice any more since they are right next to each other now. Maybe move it above on its own line?